SymHop/src/SymHop.cpp

```diff
@@ -1279,6 +1279,203 @@ QString Expression::toString() const
     return ret;
 }
 
+QString Expression::toLaTeX() const
+{
+    QString ret;
+
+    if(this->isSymbol())
+    {
+        QStringList greekSymbols = {"alpha","beta","gamma","Gamma","delta","Delta","epsilon","varepsilon","zeta","eta","theta","vartheta","Theta","iota","kappa","lambda","Lambda","mu","nu","xi","Xi","pi","Pi","rho","varrho","sigma","Sigma","tau","upsilon","Upsilon","phi","varphi","Phi","chi","psi","Psi","omega","Omega"};
+
+        if(greekSymbols.contains(mString)) {
+            ret = "\\"+mString;
+        }
+        else if(this->isNumericalSymbol()) {
+            ret = QString::number(mString.toDouble());
+        }
+        else {
+            ret = mString;
+        }
+    }
+    else if(this->isFunction())
+    {
+        QStringList builtinFunctions = {"sin","cos","tan","arcsin","arccos","arctan","cot","sec","csc","sinh","cosh","tanh","lg","ln","log","min","max"     };
+        if(mFunction == "pi" && mArguments.size() == 0) {
```

Comment:
Is pi a function ?

Reply:
Yes, in SymHop, but not in Latex. Therefore `pi()` must be converted to `\pi`.